mindmeld/core.py

```diff
@@ -229,6 +239,29 @@ def __init__(
         self._timestamp = timestamp
         self.stemmed_tokens = stemmed_tokens or tuple()
 
+    def to_cache(self):
+        return {
+            'raw_text': self.text,
+            'processed_text': self.processed_text,
+            'normalized_tokens': self._normalized_tokens,
+            'char_maps': {},
```

Comment:
Why are the char maps empty here?

Reply:
TODO: find out if char maps are used in training

Reply:
There are quite a few places that are not in the `nlp.train` flow that rely on `char_maps`. For example:
`nlp.inspect -> process_markup -> _process_annotations -> NestedEntity.from_query -> transform_span -> transform_index -> _unprocess_index -> _char_maps`